python/pythonlab/neighborhood/tests/support/constants.py

```diff
@@ -1,2 +1,4 @@
-# 2x2 grid
-SAMPLE_MAZE = '[[{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":1,"assetId":0}],[{"tileType":0,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0}]]'
\ No newline at end of file
+# 2x2 grids
+SAMPLE_MAZE = '[[{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":1,"assetId":0}],[{"tileType":0,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0}]]'
+SAMPLE_MAZE_2 = '[[{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":1,"assetId":0}],[{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0}]]'
+SAMPLE_MAZE_3 = '[[{"tileType":1,"value":3,"assetId":303},{"tileType":1,"value":1,"assetId":0}],[{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0}]]'
```

Comment:
nit: mind renaming the variables to more easily distinguish the mazes from each other (for example, looks like maze 3 has a paint bucket?)